server.js

```diff
@@ -47,8 +47,9 @@ function serveBrowserified(file, standaloneName, doBabelify) {
 }
 
 function twoBrowserified(url, file, standaloneName) {
```

Comment:
OK as you requested I created a different PR #756 for server.js changes. could you take a look there?